tools/debug/debugsymbolviewer.py

```diff
@@ -28,6 +28,7 @@
 debug = False				# Variable to print debug logs
 pc_value = 0x00000000			# Program counter value
 format_print = True
+CONFIG_PATH = '../../os/.config'
```

Comment:
instead of hardcode path can we have CONFIG_PATH like which is used in the ramdumpparser.py?

Reply:
we are also setting ```CONFIG_PATH``` in ramdumpParser.py at start.
https://github.com/Samsung/TizenRT/blob/849427eb66d1800ad584c0e57556edec8d1ad6aa/tools/trap/ramdumpParser.py#L56
Since, we need it here, then we need to define here also.

Reply:
As far as I know, the tool provides selectable options for this. 
1. use default values
2. get user input for this

Is it the same? We should concern this can be executed without source code.

Reply:
Yes, That is it. Apart from ```CONFIG_PATH```, there was one other variable ```bin_path```, which was hard-coded in debugsymbolviewer.py. I have changed it and passed both variable from ramdumpParser.py as argument.

Reply:
@neel-samsung I think you modified it to get as an new argument always. But that's not what I said.
If user inputs the path with an argument, then the script uses given path. If user does not give, then the script should use default path.

Reply:
@neel-samsung That's for user input. If the script is not for user, no need to do that like above.

Reply:
@neel-samsung Regardless of the comments above, what I think is that
we should guarantee the trap working without code build. In other words, when we get assert log and elf only, we want to run the trap without code build and on custom path (elf is not in the build/output/bin). That's why I asked you to support user input.

Reply:
@sunghan-chang The change I have made is in ```debugsymbolviewer.py``` and I thought this file will execute only when ```ramdumpParser.py``` will call internally. If that's not the case, we need to handle it. But currently, trap will run both the ways, like when user has passed the path as argument, it will take that or else default one. I haven't altered it in ```ramdumpParser.py```. 
Do we want ```debugsymbolviewer.py``` to run individually? .. Then, we need to handle it. 
https://github.com/Samsung/TizenRT/blob/c4d2e718f0cb1991d1a6393a811f2b852b9cb0d6/tools/trap/ramdumpParser.py#L55
https://github.com/Samsung/TizenRT/blob/c4d2e718f0cb1991d1a6393a811f2b852b9cb0d6/tools/trap/ramdumpParser.py#L56

Reply:
@neel-samsung Is there any usecase to run it individually? I think TRAP script is enough for user. If you think no usecase, we can keep current. No need to handle both cases (user input, default path). Please let me know your opinion.

Reply:
I also think TRAP script is enough for the user. We can keep the current change.